.devcontainer/Dockerfile

```diff
@@ -2,12 +2,7 @@
 
 # [Choice] Python version: 3, 3.9, 3.8, 3.7, 3.6
 ARG VARIANT="3"
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-${VARIANT}
```

Comment:
This container has been deprecated since last year so migrated to the new version